.coveragerc

```diff
@@ -11,4 +11,6 @@ exclude_lines =
   if __name__ == .__main__.:
 
 omit =
+  */test/*
+  */python?.?/*
```

Comment:
The commit message for this change says that `*/python?.?/*` are Python packages.  I haven't yet seen these folders in our projects.  Is this related to pyenv or virtualenv?

You should also verify whether some of the other omit lines are needed.  For example, I think `get_password` and `check_crypto_libraries` are no longer needed.

Reply:
`*/python?.?/*` excludes the parent directory of `*/site-packages/*` which also contains python modules, of which one made it into my coverage report recently, hence I added it to the omit section.

I'll remove the obsolete lines though.